bundles/SimpleBackendSearchBundle/src/Controller/SearchController.php

```diff
@@ -102,7 +103,12 @@ public function findAction(Request $request, EventDispatcherInterface $eventDisp
 
         $queryCondition = '';
         if (!empty($query)) {
-            $queryCondition = '( MATCH (`data`,`properties`) AGAINST (' . $db->quote($query) . ' IN BOOLEAN MODE) )';
+            $fullTextSearchCondition = 'MATCH (`data`,`properties`) AGAINST (' . $db->quote($query) . ' IN BOOLEAN MODE)';
```

Comment:
```suggestion
            $fullTextSearchCondition = 'MATCH (`data`,`properties`) AGAINST (' . $db->quote($query.'*') . ' IN BOOLEAN MODE)';
```
alternative appending `*` and still using the full text search?